src/Helper/NphParticipant.php

```diff
@@ -47,6 +47,16 @@ public function __isset(string $key)
         return true;
     }
 
+    public function getModule1TissueConsentStatus(): ?bool
```

Comment:
Could you take a property module1TissueConsentStatus and assign the return value of this method in parseRdrParticipant method? When you do this change you can make this method private.

`$this->module = $this-> getModule1TissueConsentStatus();`